go.sum

```diff
@@ -27,6 +27,10 @@ github.com/aws-controllers-k8s/runtime v0.0.4-0.20210224005142-96355b6c73d7 h1:H
 github.com/aws-controllers-k8s/runtime v0.0.4-0.20210224005142-96355b6c73d7/go.mod h1:xA2F18PJerBHaqrS4de1lpP7skeSMeStkmh+3x5sWvw=
 github.com/aws-controllers-k8s/runtime v0.0.5 h1:WdcnMNdgagF2MMPQRbDJ5OEzMMgHraCJqvvFj4Sx/5g=
 github.com/aws-controllers-k8s/runtime v0.0.5/go.mod h1:xA2F18PJerBHaqrS4de1lpP7skeSMeStkmh+3x5sWvw=
+github.com/aws-controllers-k8s/runtime v0.0.6-0.20210408033117-eabde5ad2882 h1:I7Y244/Ig9WqO0Jn8Cb4VSsfO3JIzf8fZS7WNfCGVpA=
+github.com/aws-controllers-k8s/runtime v0.0.6-0.20210408033117-eabde5ad2882/go.mod h1:xA2F18PJerBHaqrS4de1lpP7skeSMeStkmh+3x5sWvw=
+github.com/aws-controllers-k8s/runtime v0.0.6 h1:7sYpS+/5Rn3yAyT0eBtHWTISXxMnD+TTAaAgYRM9XOo=
+github.com/aws-controllers-k8s/runtime v0.0.6/go.mod h1:xA2F18PJerBHaqrS4de1lpP7skeSMeStkmh+3x5sWvw=
```

Comment:
Feel free to run `go mod tidy` in a future PR and remove the older runtime references in your go.sum.